runtime/src/accounts_db.rs

```diff
@@ -242,17 +262,19 @@ impl AccountsDB {
             .filter(|store| store.fork_id == fork_id)
             .cloned()
             .collect();
-        storage_maps
-            .into_par_iter()
-            .map(|storage| {
-                let accounts = storage.accounts.accounts(0);
-                let mut retval = B::default();
-                accounts
-                    .iter()
-                    .for_each(|stored_account| scan_func(stored_account, &mut retval));
-                retval
-            })
-            .collect()
+        self.thread_pool.pool.install(|| {
```

Comment:
We could also impl deref for the thread_pool such that we don't need to do thread_pool.pool ?

Reply:
Removed ParThreadPool